.github/workflows/workflow-reporter.yml

```
@@ -0,0 +1,24 @@
+workflowStat: true
```

actionlint (1.7.7)

unexpected key "workflowStat" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)